_posts/2020-09-07-witwa-nominations.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "WiTWA[+] Nominations 2020"
+date: 2020-09-07
+tags: WiTWA Tech
+author: Natalie Laing
+---
+
+![Mechanical Rock Ladies](/img/MechRockLadies.png)
+
+<br/>
+
+## My story
+
+My experience has been like that of so many women in the tech industry. At university, I could count the number of women on my course on one hand. At my first job I was the only female on a team of twelve developers.When I joined Mechanical Rock I was employee number fifteen and again I was the only woman. I thought this was normal.
+
+Over the past 2 and a half years I have seen Mechanical Rock grow both in size and diversity. We have hired some amazingly talented women and I have loved seeing how the company has evolved as each of these ladies have brought something so new and wonderful to our team. In fact, my current project is majority female and I love that I get to pair with other talented female developers every day.  
+I can honestly say that it makes a difference, and it has made Mechanical Rock an even more amazing place to work.
```

Comment:
Perhaps this could be bolded / highlighted?

Reply:
The last sentence?

Reply:
This bit
"I can honestly say that it makes a difference, and it has made Mechanical Rock an even more amazing place to work."

Reply:
Ok I have made that bold